.github/workflows/publish.yaml

```diff
@@ -0,0 +1,54 @@
+name : Publish Webhook Images
+
+on:
+  push:
+    tags:
+      - release/v*
+
+env:
+  REGISTRY: docker.io
+  REPO : rancher
+
+jobs:
+  ci :
+    uses: rancher/webhook/.github/workflows/ci.yaml@release/v0.5
```

Comment:
`github.base_ref` returns the branch in `ci.yaml`, I'll need to verify this works in the other workloads (see Michael's comment on further testing of the other workloads still needed).

Reply:
Here's what I've found about re-using a workflow that's part of the current repo. Have a look at https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow:
> If you use the second syntax option (without {owner}/{repo} and @{ref}) the called workflow is from the same commit as the caller workflow.

This would look like:

```yaml

jobs:
  ci :
    uses: ./.github/workflows/ci.yml
```

This way no need to fiddle with any branch names/commits,etc.